nvm.sh

```diff
@@ -554,7 +553,7 @@ ${NVM_LS_REMOTE_POST_MERGED_OUTPUT}" | nvm_grep -v "N/A" | command sed '/^$/d')"
     return 3
   fi
   nvm_echo "${VERSIONS}"
-  return $NVM_LS_REMOTE_EXIT_CODE || $NVM_LS_REMOTE_IOJS_EXIT_CODE
+  return ${NVM_LS_REMOTE_EXIT_CODE} || ${NVM_LS_REMOTE_IOJS_EXIT_CODE}
```

Comment:
i'm confused why you'd use curly braces in a non-quoted string

Reply:
Just didn't see the rule of braces here, for example:
Line 264(NVM_CD_FLAGS):
`NVM_DIR="$(nvm_cd ${NVM_CD_FLAGS} "$(dirname "${NVM_SCRIPT_SOURCE:-$0}")" > /dev/null && \pwd)"`

Line 2627:
`case ${line} in `

Maybe I should remove them together?

Reply:
The pattern I've tried to follow is, always use the braces when a variable is referenced inside a quoted string, and never use them outside a quoted string. I'm sure it's not consistent, though.

If there's a good reason to use a different style, I'd be interested to hear it - otherwise it'd be good to stick to that.